src/codegeneration/TemplateLiteralTransformer.js

```diff
@@ -198,6 +200,21 @@ export class TemplateLiteralTransformer extends TempVarTransformer {
     return ParseTreeTransformer.prototype.transformFunctionBody.call(this, tree);
   }
 
+  transformNewExpression(tree) {
+    if (tree.operand) {
+      let operand = tree.operand;
+      do {
+        // If operand contains a tagged TemplateLiteral, parenthesize the entire
+        // operand.
+        if (operand.type === TEMPLATE_LITERAL_EXPRESSION && operand.operand !== null) {
+          tree = createNewExpression(createParenExpression(tree.operand), tree.args);
```

Comment:
This works but in general it is better to use the constructors so that the location is not lost.

```js
tree = new NewExpression(tree.location,
    createParenExpression(tree.operand), tree.args);
```

since the paren expression is generated code there is no location for it so either `createParenExpression(expr)` or `new ParenExpression(null, expr)` is fine.